STUDIO/org.openl.rules.jackson/src/org/openl/rules/serialization/ObjectMapperCache.java

```diff
@@ -0,0 +1,72 @@
+package org.openl.rules.serialization;
+
+/*-
+ * #%L
+ * OpenL - STUDIO - Jackson
+ * %%
+ * Copyright (C) 2016 - 2021 OpenL Tablets
+ * %%
+ * See the file LICENSE.txt for copying permission.
+ * #L%
+ */
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.WeakHashMap;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ObjectMapperCache {

```

Comment:
Why OpenL should care about caching of ObjectMapper? I believe it must be covered by the client integration. If he wants to cache it, so he must care about it.
Any unused classes in OpenL confuse and are wanted to be deleted :)

Reply:
It is used in OpenL Insight what is also OpenL :)

Reply:
Why cannot we place this logic there?
For example, I'm deleting classes that are not in use in the current project, also we will have sonar issues if the class is not in use.

Reply:
This logic must be available in WebStudio too